### PR TITLE
Adding RLN keystore

### PR DESCRIPTION
**Pull request by apentori**
Linked to : 
* https://github.com/status-im/infra-waku/issues/19

### CODE REVIEW

**Review by yakimant — APPROVED** [review]
All good, but I would rather run generate script in Ansible no runtime.

File: templates/run-node.sh.j2 [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+{# WARNING: Keep sensitive information in run-node.sh. #}
+{% if nim_waku_node_key is defined %}
+      --nodekey={{ nim_waku_node_key }}
+{% endif %}
+{% if "store" in nim_waku_protocols_enabled %}
+      --store-message-db-url='{{ nim_waku_store_message_db_url }}'
+{% endif %}
+{% if nim_waku_rln_keystore_active %}
+      --rln-relay-cred-password='{{ nim_waku_rln_cred_password | mandatory }}'
+{% endif %}
```
Comment (jakubgs) [inline templates/run-node.sh.j2:33]: This is wrong. Those are argument that should be in the Docker compose. This is why there is a distinction between encrypoint and command. This is why God invented `${@}`.

File: templates/docker-compose.yml.j2 [hunk 2]
```diff
@@ -20,6 +20,7 @@ services:
     volumes:
       - '{{ nim_waku_node_conf_path }}:/conf'
       - '{{ nim_waku_node_db_path }}:/data'
+      - '{{ nim_waku_cont_vol }}/run-node.sh:/opt/run-node.sh:Z'
```
Comment (jakubgs) [inline templates/docker-compose.yml.j2:23]:
```suggestion
      - '{{ nim_waku_cont_vol }}/run-node.sh:/opt/entrypoint.sh:Z'
```

File: tasks/config.yml [hunk 3]
```diff
@@ -26,3 +26,11 @@
     owner: 'dockremap'
     group: 'docker'
     mode:  0755
+
+- name: Create script to run node
+  template:
+    src: 'run-node.sh.j2'
+    dest: '{{ nim_waku_cont_vol }}/run-node.sh'
```
Comment (jakubgs) [inline tasks/config.yml:33]:
```suggestion
- name: Create node Docker entrypoint
  template:
    src: 'entrypoint.sh.j2'
    dest: '{{ nim_waku_cont_vol }}/entrypoint.sh'
```

**Review by jakubgs — CHANGES_REQUESTED** [review]

File: templates/docker-compose.yml.j2 [hunk 4]
```diff
@@ -34,12 +35,18 @@ services:
       test: ["CMD", "wget", "-qO-", "http://127.0.0.1:{{ nim_waku_metrics_port }}/health"]
       interval: 30s
       retries: 3
+    entrypoint: /opt/entrypoint.sh
     command: |
       --config-file=/conf/config.toml
-{# WARNING: Keep sensitive information in Docker Compose file. #}
+{# WARNING: Keep sensitive information in run-node.sh. #}
```
Comment (Ivansete-status) [inline templates/docker-compose.yml.j2:41]: nitpick comment. On the other hand, it seems that we are keeping sensitive info in both the script and the docker-compose.yml file. I think is interesting to keep it in the script file only, to simplify it a little.
```suggestion
{# WARNING: Keep sensitive information in entrypoint.sh. #}
```

**Review by Ivansete-status — APPROVED** [review]
LGTM! Thanks for it! :100: 
I just added a nitpick comment

File: templates/docker-compose.yml.j2 [hunk 5]
```diff
@@ -34,12 +35,18 @@ services:
       test: ["CMD", "wget", "-qO-", "http://127.0.0.1:{{ nim_waku_metrics_port }}/health"]
       interval: 30s
       retries: 3
+    entrypoint: /opt/entrypoint.sh
     command: |
       --config-file=/conf/config.toml
-{# WARNING: Keep sensitive information in Docker Compose file. #}
+{# WARNING: Keep sensitive information Docker Compose file. #}
```
Comment (jakubgs) [inline templates/docker-compose.yml.j2:41]:
```suggestion
{# WARNING: Keep sensitive information in Docker Compose file. #}
```

**Review by jakubgs — APPROVED** [review]
Nice.

**Pull request merged**